strawberry_django_plus/settings.py

```diff
@@ -27,6 +27,7 @@ class Config:
         default_factory=lambda: ["Input", "Partial"],
     )
     FIELDS_USE_GLOBAL_ID: bool = dataclasses.field(default=True)
+    GENERATE_ENUMS_FROM_CHOICES: bool = dataclasses.field(default=True)
```

Comment:
I think it is best to have this as `False` by default since it is a breaking change

```suggestion
    GENERATE_ENUMS_FROM_CHOICES: bool = dataclasses.field(default=False)
```